### ISSUE TITLE
Parallelize python-inspector during analyzer execution

### ISSUE THREAD

**Issue by bennati** [oss-review-toolkit/ort#6206]
Python inspector runs on a single core, parallelizing its execution could speed up large python projects.

**Comment by sschuberth** [comment]
As discussed in the [community meeting](https://github.com/oss-review-toolkit/ort/wiki/ORT-Community-Meeting#2023-04-27), the ORT analyzer does not run any single of its analyzers in parallel currently (only analyzers for different definition files are run in parallel). In general, it's not in the scope of the ORT analyzer to implement parallelism when running an analyzer of a specific type; instead, that is the responsibility of the analyzer implementation, i.e. in this case the python-inspector. So I'm closing this in favor of the upstream python-inspector issue. When updating to a python-inspector version that has this feature implemented, we'll take care of making use of it in ORT.
